### DISCUSSION TITLE
supareviews

### DISCUSSION THREAD

**Discussion by unofficialdxnny** [supabase/supabase#3737] [Questions] [answered]
Hey so i made a discord bot called `infinity` and obviously i need a website for it. Instead of just having a simple invite and features page. this is the reviews page i want to use https://codepen.io/entbit/pen/XWjjjbJ , in the js it has these lines that are used to connect to the base to store the reviews
```js
// Supabase initialization
const supaURL = '';
const supaKEY = '';
const supabase = createClient(supaURL, supaKEY);
```

idk where i can find the `supaURL` supaKEY`
please help

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.io/docs/reference/javascript/initializing

They are under settings, then api in the UI you get when you create an account.
